src/app/pages/server-details/server-details.component.ts

```
@@ -1372,16 +1379,62 @@ export class ServerDetailsComponent implements OnInit, OnDestroy {
     }
   }
 
+  generateStressNGChart(benchmark_id: string, scaleField: string, isLineChart: boolean = true) {
+    const dataSet = this.benchmarksByCategory?.find(x => x.benchmark_id === benchmark_id);
+
+    if(dataSet && dataSet.benchmarks?.length) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)